frontend/apps/marketing/src/config/stage.ts

```diff
@@ -1,10 +1,11 @@
-export type Stage = 'development' | 'test' | 'production';
+export type Stage = 'development' | 'pr' | 'test' | 'production';
```

Comment:
What does `pr` stand for?

Reply:
Pull Request (when the code runs in a pull request)

Reply:
Ok cool! That's what I thought but wanted to confirm.